elide-core/src/main/java/com/yahoo/elide/security/permissions/ExpressionResult.java

```diff
@@ -5,54 +5,23 @@
  */
 package com.yahoo.elide.security.permissions;
 
-import lombok.Getter;
-
 /**
  * Expression results.
  */
-public class ExpressionResult {
-    @Getter private final Status status;
-    @Getter private final String failureMessage;
-
-    public static final ExpressionResult PASS_RESULT = new ExpressionResult(Status.PASS);
-    public static final ExpressionResult DEFERRED_RESULT = new ExpressionResult(Status.DEFERRED);
+public enum ExpressionResult {
+        PASS("PASSED"),
+        FAIL("FAILED"),
+        DEFERRED("WAS DEFERRED"),
+        UNEVALUATED("WAS UNEVALUATED");
 
-    /**
-     * Result status.
-     */
-    public enum Status {
-        PASS,
-        FAIL,
-        DEFERRED
-    }
-
-    /**
-     * Constructor.
-     *
-     * @param status Status to return
-     * @param failureMessage Message associated with failure
-     */
-    public ExpressionResult(final Status status, final String failureMessage) {
-        this.status = status;
-        this.failureMessage = failureMessage;
-    }
+    private String name;
 
-    /**
-     * Constructor.
-     *
-     * @param status Status to return
-     */
-    public ExpressionResult(final Status status) {
-        this(status, null);
+    ExpressionResult(String name) {
+        this.name = name;
     }
 
     @Override
     public String toString() {
-        StringBuilder sb = new StringBuilder();
-        sb.append(status);
-        if (failureMessage != null) {
-            sb.append(": ").append(failureMessage);
-        }
-        return sb.toString();
+        return name;
```

Comment:
I recommend using [Jansi](https://github.com/fusesource/jansi) here. It would be great to get some colors based on the message. Perhaps in the constructor for the enum, a color code could be included and this statement would be come:

```java
return ansi().fg(colorCode).a(name).reset();
```